### PR TITLE
New animations

### PR DESCRIPTION
**Pull request by nildo**
Added new sprites and animations for the robot and asteroids. These modifications need the fix in the resource loader.

### CODE REVIEW

File: src/lesson/lesson01.js [hunk 1]
```diff
@@ -161,12 +161,14 @@ Lesson01ExerciseStepPlayer.prototype = {
...
       for (var obstacle_pos in this.game.obstacles[i]) {
-        var obstacle = new Animator.CircleElement(
+        var obstacle = new ElementFactories.createAsteroid(
             'o' + i + obstacle_pos, grid_cell_size / 2);
         obstacle.x = (0.5 + this.game.obstacles[i][obstacle_pos]) *
           grid_cell_size;
         obstacle.y = (0.5 + (this.game.n_rows - 1 - i)) * grid_cell_size;
         this._animator.addElement(obstacle);
+        var asteroid_number = (this.game.n_rows * i + obstacle_pos) % 8;
+        this._animator.addAnimation(obstacle.createAnimation('asteroid_' + asteroid_number, 0, 1, 1));
```
Comment (vrod) [inline src/lesson/lesson01.js]: nitpick: break line, so every line contains at most 80 characters

**Comment by vrod** [comment]
@nildo: For some reason, when I try to run the code in tha latest version of your revision, there is a blank screen and nothing shows in the console. I will investigate it further in the future, but for now I will put this on hold. Does it currently work for you, as it is?

**Comment by nildo** [comment]
It's because of the resource loader bug. @gpoesia fixed it here: https://github.com/gpoesia/comp4kids/pull/23 We should merge it.

**Comment by vrod** [comment]
@nildo: Alright, it worked after applying #23.

To keep things standardized, I suggest you squash/rebase your 4 commits into a single commit, and always use the imperative form in the commit message (e.g., prefer "fix" over "fixing" or "fixed", "add" over "adding" or "added"). Let me know if you need help with that.

**Comment by gpoesia** [comment]
Rebased and merged (ed199cec59854ee5581772db89d85df5b20f89a8)

**Pull request closed without merging**